services/certificate-api/src/utilities/cord.js

```diff
@@ -0,0 +1,105 @@
+// src/utilities/cord.js
+// CORD-related functions.
+
+import * as cord from '@cord.network/sdk'
+
+import { CORD_CHAIN_ADDRESS } from '../../config/index.js'
+
+// First, connect to the chain.
+await cord.init({ address: CORD_CHAIN_ADDRESS })
+
+// Retrieve the identities used to create schemas and credentials.
+// TODO: `Alice` works, but a newly created account doesn't. Why?
+const signer = cord.Identity.buildFromURI('//Alice', {
+	signingKeyPairType: 'sr25519',
+})
+// Create the schema and register it once `createClient` is called.
+const schemaContent = {
```

Comment:
This is a dummy schema for the demo. Ideally, the schema associated with the credential should be passed here.

Reply:
I guess here, we use schemas defined as per the workflow, (like teachers.json and students.json) etc. The same should be passed for credential, so the content validation also happens based on the schema.